f2f-ipv-stub/template.yaml

```diff
@@ -42,8 +42,10 @@ Mappings:
   Configuration:
     dev:
       IPVStubID: "5C584572"
+      JWKS_URI: "https://api-f2f-cri-api.review-o.dev.account.gov.uk"
     build:
       IPVStubID: "BD7B2A5D"
+      JWKS_URI: "https://api.review-0.build.account.gov.uk"
```

Comment:
it should be JWKS_URI: "https://api.review-o.build.account.gov.uk"

Reply:
Done thanks!